front/src/modules/trainschedule/components/DriverTrainSchedule/DriverTrainScheduleStop.tsx

```diff
@@ -52,7 +52,10 @@ export default function DriverTrainScheduleStop({ stop, idx, train }: Props) {
       <td className="d-flex justify-content-center">
         <div>{Number.isInteger(pk) ? `${pk}.0` : pk}</div>
       </td>
-      <td>{stop.name || 'Unknown'}</td>
+      <td>
+        {stop.name || 'Unknown'}&nbsp;
+        {stop.ch === '00' || !stop.ch ? 'BV' : `${stop.ch}`}
```

Comment:
Do we want to display `BV` by default? I didn't find the information in the ticket and have not enough background to understand it 😄

Reply:
you're right, it was asked at first but removed since yersterday

Reply:
you're right, it was asked at first but removed since yersterday

Reply:
I wrote on the issue that this was out of the scope for now. I kept it as ~~mistaken text~~ to keep track of this change, but the criteria should still be visible

Reply:
BV/00 should not be displayed IMO